.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+Doc/library/hashlib.rst        @tiran
```

Comment:
Since this file might grow a lot, we should probably have a comment per section to say what the rules are meant for, e.g. `# Hashing`.